src/mte_intro.adoc

```diff
@@ -1,79 +1,77 @@
 [[intro]]
 == Introduction
 
-Significant production deployed software is written in languages which provide
-powerful primitives (like C/C++) to software developers to manage memory.
+Significant production software is written in languages which provide
+powerful memory management primitives (like C/C++).
 Software is written in collaborative fashion and on tight deadlines and bugs do
-get introduced in software development. Such bugs sometimes can lead to issues
-like information disclosures, memory leak or memory corruptions commonly termed
-as `memory safety` issues. Underlying cause of memory safety issues can be
-anything but eventually such issues surface as invalid memory accesses.
+get introduced. Such bugs can lead to issues like information disclosures,
+memory leak or memory corruptions commonly termed as "memory safety" issues.
```

Comment:
Lack of Oxford comma makes it unclear what is considered a memory safety issue.

Mentioning memory leaks is extraneous. They're not memory safety issues and memory tagging won't usefully fix them.